tests/integration/servicemesh/service_mesh_setup_features_int_test.go

```diff
@@ -535,6 +546,14 @@ func newDSCInitializationSpec(ns string) *dscv1.DSCInitializationSpec {
 	return &spec
 }
 
+func newDefaultOrigin() featurev1.Origin {
```

Comment:
Maybe we can extract this and the other `newOrigin` to somewhere where we share test fixtures and use it across instead of duplicating the code? I am also wondering if `Default` is self-explanatory here.

Reply:
[103593f](https://github.com/maistra/opendatahub-operator/pull/17/commits/103593f01d204ded9ae413064bddcd5b8e9a5131) done here